user/enterprise/worker-configuration.md

```diff
@@ -118,12 +118,10 @@ The Worker comes configured with the RAM defaulted to 4G. If you want to
 change it you can add the following. To completely disable it have the
 value set to 0.
 
-```
-      export TRAVIS_WORKER_DOCKER_MEMORY=4G
-```
+```sh
+export TRAVIS_WORKER_DOCKER_MEMORY=4G
 
-```
-      export TRAVIS_WORKER_DOCKER_MEMORY=0
+export TRAVIS_WORKER_DOCKER_MEMORY=0
```

Comment:
We should probably leave this as two distinct code blocks to reinforce that it is an either / or options?

Reply:
Done!